start-build-env.sh

```diff
@@ -28,7 +28,7 @@ cd "$(dirname "$0")"
 : ${AMBARI_DIR:=$(pwd -P)}
 
 # Maven version
-: ${MAVEN_VERSION:=3.3.9}
```

Comment:
Is it really meant to be included into a patch?

Reply:
originally i wanted it in a different patch, but it's not really a problem if it goes with this one